docs/tutorials/troubleshooting/deploy-vm.md

```diff
@@ -0,0 +1,54 @@
+  Change Block Tracking:  false
+  Conditions:
+    Last Transition Time:  2022-07-06T00:43:47Z
+    Status:                Unknown
+    Type:                  GuestCustomization
+    Last Transition Time:  2022-07-06T00:42:41Z
+    Status:                True
+    Type:                  VirtualMachinePrereqReady
+    Last Transition Time:  2022-07-06T00:43:47Z
+    Message:               VMware Tools is not running
+    Reason:                VirtualMachineToolsNotRunning
+    Severity:              Error
+    Status:                False
+    Type:                  VirtualMachineTools
+...
+```
+
+## Error and fix
+This section focuses on leveraging the information provided by the `VM.Status.Conditions` field to diagnose the root cause of errors and provide solutions. By analyzing which conditions showed `Status` as `False`/`Unknown` and `Reason`, you can troubleshoot issues depending on below scenarios.
+
+### VirtualMachinePrereqReady
+- Reason `VirtualMachineClassNotFound`/`VirtualMachineClassBindingNotFound`: this means that the VM class has not been associated to the Supervisor namespace.
+    - **Fix**: Attach VM Class to your namespace. Refer to [VMClass](https://vm-operator.readthedocs.io/en/stable/concepts/workloads/vm-class/) for details.
+- Reason `VirtualMachineImageNotFound`: this means that the VirtualMachine Image doesn't exist in the supervisor cluster, which means that this VM image doesn't exist in the vCenter, or the content library which it belongs to has not been associated to any of the supervisor namespaces. 
+    - **Fix**: Upload mentioned VM Image to content library and associate content library to your namespace. Refer to [VM Image](https://vm-operator.readthedocs.io/en/stable/concepts/images/) for details.
+- Reason `ContentSourceBindingNotFound`: this means that the content library which contains the specified VM Image has not been associated to the Supervisor namespace. 
+    - **Fix**: Associate content library to your namespace. Refer to [VM Image](https://vm-operator.readthedocs.io/en/stable/concepts/images/) for details.
+
+### GuestCustomization/VirtualMachineTools
+- Reason `GuestCustomizationFailed`/`VirtualMachineToolsNotRunning`:
+This could be happening when using transport `OvfEnv/ExtraConfig` to deploy a VM. These two transports use VMTools for network configuration, using incompatible virtual machine images could cause race with cloud-init user-data configuration. 
+    - **Fix**: use transport `CloudInit`. Refer to [Deploy with CloudInit](https://vm-operator.readthedocs.io/en/stable/tutorials/deploy-vm/cloudinit/) for details.
+
+## Best practice
+- Create a namespace
+- Make sure VM Image is uploaded to content library
+- Make sure content library is associated to the namespace
+- Make sure storage class is specified in the namespace
+- Make sure VM Class is attached to the namespace
+- Make sure appropriate transport is chosen
+- Deploy VM
```

Comment:
These steps appear to be useful for troubleshooting VM deployment failures. I would suggest moving these items above and including kubectl command and output examples for each section. Feel free to take a look at the [Get a Console Session](https://vm-operator.readthedocs.io/en/latest/tutorials/troubleshooting/get-console-session/) guide.